labelbox/client.py

```diff
@@ -55,7 +55,8 @@ def __init__(self,
                  api_key=None,
                  endpoint='https://api.labelbox.com/graphql',
                  enable_experimental=False,
-                 app_url="https://app.labelbox.com"):
+                 app_url="https://app.labelbox.com",
+                 rest_endpoint="https://api.labelbox.com/api/v1"):
```

Comment:
Added a new `rest_endpoint` param to the constructor that points to the base url for the new api rest endpoints.